packages/react/src/components/Modal/ModalFooter/ModalFooter.module.css

```diff
@@ -2,6 +2,8 @@
   display: flex;
   align-items: center;
   gap: var(--fds-spacing-4);
-  padding: var(--fds-spacing-3) var(--fds-spacing-6);
-  padding-bottom: 0;
+  padding-left: var(--fds-spacing-3);
+  padding-right: var(--fds-spacing-3);
+  padding-top: var(--fds-spacing-6);
+  padding-bottom: var(--fds-spacing-6);
```

Comment:
These don't need to be separate - we can still use `padding: var(--fds-spacing-3) var(--fds-spacing-6)`, as long as we don't set `-bottom: 0`.
It looks like the padding is the wrong way around as well

Reply:
<img width="554" alt="image" src="https://github.com/digdir/designsystem/assets/14057205/af0d507b-7a9e-4bc2-bf56-6ff571215fde">

Reply:
I know, but I think it's clearer which paddings are vertical and horizontal when explicitly stating the left-padding, etc.

Although, I see the paddings are switched yeah.

Reply:
@mimarz thoughts?